src/oatpp-zlib/Processor.cpp

```diff
@@ -63,6 +63,11 @@ DeflateEncoder::DeflateEncoder(v_buff_size bufferSize, bool gzip, v_int32 compre
 
 }
 
+DeflateEncoder::~DeflateEncoder() {
+  v_int32 res = deflateEnd(&m_zStream);
+  OATPP_ASSERT(res == Z_OK);
```

Comment:
Let's remove the assert.
Let's just log the error instead:

```cpp
OATPP_LOGE("[oatpp::zlib::DeflateEncoder::~DeflateEncoder()]", "Error. Failed call to 'deflateEnd()'.")
```

Reply:
Thanks for your suggestion. I have modified the code. 
The same problem was found in the decoder and fixed. 
Please review, thank you!